src/main/java/sopt/makers/authentication/adapter/in/web/controller/user/UserApi.java

```diff
@@ -29,11 +34,16 @@ ResponseEntity<BaseResponse<?>> updateUserProfile(
       @PathVariable Long userId,
       @Valid @RequestBody UserRequest.UserProfileInfo userProfileInfo);
 
-  ResponseEntity<BaseResponse<?>> getUserProfileByActivity(
+  ResponseEntity<BaseResponse<?>> getUserProfileByFilters(
       @RequestHeader(API_KEY_HEADER) String apiKey,
       @RequestHeader(SERVICE_NAME_HEADER) String serviceName,
       @RequestParam(required = false) Integer generation,
-      @RequestParam(required = false) Part part);
+      @RequestParam(required = false) Part part,
+      @RequestParam(required = false) String name,
+      @RequestParam(required = false) Team team,
+      @RequestParam(defaultValue = DEFAULT_OFFSET) int offset,
+      @RequestParam(defaultValue = DEFAULT_LIMIT) @Positive @Max(MAX_LIMIT) int limit,
```

Comment:
```suggestion
    @RequestParam(defaultValue = DEFAULT_OFFSET) @Min(0) int offset,
    @RequestParam(defaultValue = DEFAULT_LIMIT) @Min(1) @Max(MAX_LIMIT) int limit,
```

offset이랑 limit에 @Min, @Max  설정해줘도 좋을 것 같아요~!